File: EvenOdd.java
import java.util.Scanner;
public class OddorEven{
    static void find(int a)
    {
        if(a%2 == 0)
        {
            System.out.println("The value "+ a +" is a even number.");
        }
        else
        {
            System.out.println("The number "+ a +" is a odd number.");
        }
    }
    public static void main(String[] args){
        Scanner input = new Scanner(System.in);
        int a;
        System.out.print("Enter the value of a : ");
        a = input.nextInt();
        find(a);
    }
}

File: EvenOdd.cpp
#include<iostream>
using namespace std;
void check(int num)
{
    if(num==0)
    {
        cout<<"The given number "<<num<<" is neither even nor odd"<<endl;
    }
    else if(num>0)
    {
        if(num%2==0)
        {
            cout<<"The given number "<<num<<" is an even number"<<endl;
        }
        else
        {
            cout<<"The given number "<<num<<" is an odd number"<<endl;
        }
    }
    else
    {
        cout<<"Please enter a positive number!"<<endl;
    }
}
int main()
{
    int num;
    cout<<"Enter a positive integer : ";
    cin>>num;
    check(num);
}